### ISSUE TITLE
Trainer Screen, Cal % values.

### ISSUE THREAD

**Issue by sloped-soarer** [Ingwie/OpenAVRc_Dev#35]
The bottom line of the Trainer Screen displays the % values for PPM pulses. With a Multiplier of 1.0, this appears to be scaled to a standard ppm range of 1500us +- 500us  e.g. 2000us = 100%.
I suggest the scaling by default should be for the Openavrc PPM range of 1500 +- 512us. e.g. 2012us = 100%.
This might make more sense when looking at the servo output screen as currently 100% = 1988us.
8)

**Comment by Ingwie** [comment]
Hi my friend, Ok it is a good modification.
Thanks

**Comment by testeur68** [comment]
Hi Ritchie, I use this fonction every week for training. It's a good idea to standardise channel values. Best regards, testeur_68Envoyé depuis mon téléphone Huawei-------- Message original --------De : sloped-soarer <notifications@github.com>Date : mer. 5 août 2020 à 18:00À : Ingwie/OpenAVRc_Dev <OpenAVRc_Dev@noreply.github.com>Cc : Subscribed <subscribed@noreply.github.com>Objet : [Ingwie/OpenAVRc_Dev] Trainer Screen, Cal % values. (#35)
The bottom line of the Trainer Screen displays the % values for PPM pulses. With a Multiplier of 1.0, this appears to be scaled to a standard ppm range of 1500us +- 500us  e.g. 2000us = 100%.
I suggest the scaling by default should be for the Openavrc PPM range of 1500 +- 512us. e.g. 2012us = 100%.
This might make more sense when looking at the servo output screen as currently 100% = 1988us.
8)

**Comment by sloped-soarer** [comment]
Hello again. Well I never stop learning !.
So looking at the code i can see why 1500us +-500us  servo range was chosen to be +-100%. The integer mathematics is simple.
However 500us / 512us = 97.65625%. So trying  to solve this using integer maths and using a 16bit signed variable range was like trying to find the nearest fraction for PI.
Anyhow multiplying 500us by 41/21 gives 976.19047619 which is close enough, just below and displays correctly to 0.1%.
Please note, I am only proposing to change the displayed percentage.

To be honest, I think the trainer screen is a bit weird. 
I did not realise that a long press of the enter button when on the "Cal" actually trimmed for the ppm centres being offset.
Also i think the multiplier is too coarse e.g. 10% steps whereas the Student weights for Ele, Ail ... are 1% steps.

Also i cannot understand why we throw away some of the resolution e.g. We read ppm values to 0.5us resolution, but instead we use 1us steps.

I will test this change a bit more before committing a change unless there are other comments.

**Comment by Ingwie** [comment]
Thanks, I'm busy on another part of the code.
Do what you think is right, you have my full confidence.
